src/renderer/components/ft-create-playlist-prompt/ft-create-playlist-prompt.vue

```diff
@@ -15,13 +15,24 @@
         :value="playlistName"
         :maxlength="255"
         class="playlistNameInput"
-        @input="(input) => playlistName = input"
+        @input="handlePlaylistNameInput"
         @click="createNewPlaylist"
       />
     </ft-flex-box>
+    <ft-flex-box v-if="playlistNameBlank">
+      <p>
+        {{ $t('User Playlists.SinglePlaylistView.Toast["Playlist name cannot be empty. Please input a name."]') }}
+      </p>
+    </ft-flex-box>
+    <ft-flex-box v-if="playlistWithNameExists">
+      <p>
+        {{ $t('User Playlists.CreatePlaylistPrompt.Toast["There is already a playlist with this name. Please pick a different name."]') }}
+      </p>
+    </ft-flex-box>
     <ft-flex-box>
       <ft-button
         :label="$t('User Playlists.CreatePlaylistPrompt.Create')"
+        :disabled="playlistPersistenceDisabled"
```

Comment:
**thought/question:** I have no idea how Vue knows to make the underlying button disabled when there doesn't seem to be a prop for this in our `ft-button` class, but it does.

Reply:
Vue calls that feature "fallthrough attributes", if the component doesn't have a prop for the attribute, it adds the attribute to the root/top-level HTML element inside that component.

Vue 3 has an entire docs page on it, but it also describes Vue 3 specific behaviour so it's not that useful for us yet, Vue 2's docs only have an offhand mention of it (https://v2.vuejs.org/v2/api/#inheritAttrs).